sdk/core/azure-core/src/main/java/com/azure/core/credentials/BasicAuthenticationCredential.java

```diff
@@ -44,6 +44,7 @@ public BasicAuthenticationCredential(String username, String password) {
         String credential = username + ":" + password;
         String encodedCredential;
         try {
+            // Please change the error message everytime you update the named charset.
```

Comment:
I don't understand what this message is saying to future developers. What error message? I would rather not have this kind of message left in code comments unless it is much clearer about what needs to happen.